### PR TITLE
fix the "Frequently changing state should not be directly read in com…

### PR DESCRIPTION
**Pull request by kazemcodes**
fix 
image

### CODE REVIEW

File: lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt [hunk 1]
```diff
@@ -74,225 +74,228 @@ fun LazyColumnScrollbar(
...
+    thumbSelectedColor: Color = Color(0xFF5281CA),
+    thumbShape: Shape = CircleShape,
+    selectionMode: ScrollbarSelectionMode = ScrollbarSelectionMode.Thumb,
+    indicatorContent: (@Composable (index: Int, isThumbSelected: Boolean) -> Unit)? = null,
 ) {
-	val coroutineScope = rememberCoroutineScope()
 
-	var isSelected by remember { mutableStateOf(false) }
+    val firstVisibleItemIndex =
+        remember { derivedStateOf { listState.firstVisibleItemIndex } }
```
Comment (nanihadesuka) [inline lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt]: I don't see how this is going to improve recomposition count but anyway, its cool too. If you have time and really want to test performance, in the last android studio (electric eel) you can count the number of recompositions :=)

**Comment by nanihadesuka** [comment]
@kazemcodes remove all the white space changes (use tabs), Then I will merge.

**Comment by kazemcodes** [comment]
my Idea setting is a little bit different from yours, I change the setting and it didn't seems to fix this issue,
and the correct format for indent is using spaces instead of Indent according to kotlin docs
image

**Comment by kazemcodes** [comment]
I also found that there is small issue in LazyColumn performance, I'm not sure where this issue comes from, but this issue can slow down the scrolling and cause lags during the usage

**Comment by nanihadesuka** [comment]
Will clean up the whitespace.

**Comment by nanihadesuka** [comment]
Ok, my bad. Seems I had tabs in one file and spaces in another.

**Pull request merged**